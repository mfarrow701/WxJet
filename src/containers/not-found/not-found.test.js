import React from 'react';
import {shallow} from 'enzyme';
import NotFound from './not-found';

describe('Not found container', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<NotFound/>);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the not-found container after load', () => {
        expect(component).toMatchSnapshot();
        expect(component.find('.Not-Found h1').text()).toEqual('Not Found!');
    });
});
