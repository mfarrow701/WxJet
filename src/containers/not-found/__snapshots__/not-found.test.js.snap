// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found container should render the not-found container after load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1>
        Not Found!
</h1>,
      "className": "Not-Found",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Not Found!",
      },
      "ref": null,
      "rendered": "Not Found!",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1>
          Not Found!
</h1>,
        "className": "Not-Found",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Not Found!",
        },
        "ref": null,
        "rendered": "Not Found!",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
